migrations/20241010142723_add_slug_field_and_anthropic_enum.sql

```diff
@@ -0,0 +1,4 @@
+-- Add value to enum type: "model_engines"
+ALTER TYPE "public"."model_engines" ADD VALUE 'ANTHROPIC';
+-- Modify "bots" table
+ALTER TABLE "public"."bots" ADD COLUMN "slug" character varying(255) NULL;
```

Comment:
Does this migration file handle existing entries by allowing the column to be null, or is it genuinely optional?

Reply:
Yes, this migration handles existing entries by allowing the column to be NULL for now. Later, once we auto-generate the slug, I’ll change it to NOT NULL.